node/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "log4js": "~0.6.9",
     "bower": "~1.3.12",
-    "lodash": "~2.2.1"
+    "lodash": "~2.2.1",
+    "bower-config": "^0.5.2"
```

Comment:
Please remove this bower-config dependency. It is not required right now, the API that it exposes is incomplete and it will add more dependencies that then I need to package.